main.py

```
@@ -1538,37 +1538,47 @@ async def callback_router(update: Update, context: ContextTypes.DEFAULT_TYPE):
             [InlineKeyboardButton("⚙️ Auth Groups", callback_data="manage_auth"),
              InlineKeyboardButton("👨‍💻 Debuggers", callback_data="manage_debug")],
-            [InlineKeyboardButton("💤 Sleep Mode", callback_data="toggle_sleep")]
+            [InlineKeyboardButton("💤 Sleep Mode", callback_data="toggle_sleep")],
+            [CLOSE_BUTTON]
         ]
         reply_markup = InlineKeyboardMarkup(keyboard)
+        await query.answer()
         try:
             antigravity_mode = "ON" if chat_id in antigravity_chats else "—"
             alchemy_mode = "ON" if chat_id in alchemy_chats else "—"
             dashboard_mode = "ON" if chat_id in dashboard_chats else "—"
```

Ruff (0.15.12)

[error] Undefined name `dashboard_chats`

(F821)